jina/docker/hubio.py

```diff
@@ -627,12 +627,14 @@ def _get_revised_dockerfile(self, dockerfile_path: str, manifest: Dict) -> str:
         # modify dockerfile
         revised_dockerfile = []
         with open(dockerfile_path) as fp:
-            for l in fp:
+            _last_from_ln = None
+            for _ln, l in enumerate(fp):
```

Comment:
Just curious about what does _ln and l stand for(like lines, labels?) here?

Reply:
i used it for `line_numbers` :)